Source/Core/DolphinTool/DolphinTool.exe.manifest

```diff
@@ -21,9 +21,6 @@
 </dependency>
 <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
   <application>
-    <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}"/>
-    <supportedOS Id="{35138b9a-5d96-4fbd-8e2d-a2440225f93a}"/>
-    <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}"/>
```

Comment:
Shouldn't DolphinTool still work on Windows 7 for the time being?

Reply:
Oh true, although I don't see much value in keeping support for it-- it'd be better for people to just use a copy from a dolphin fork.

Reply:
I would very much not like to give fuel to forks just for the sake of getting to delete three lines. (Not that I think someone is actually going to fork Dolphin just to run DolphinTool on Windows 7...)

Reply:
Fair enough, I'll make a comment instead.